projects/CSM_BeEST.yaml

```diff
@@ -0,0 +1,12 @@
+Name: CSM_BeEST
```

Comment:
In the past you've used "Mines_" as the prefix for Colorado School of Mines projects.